README.md

```diff
@@ -14,25 +14,168 @@ More information can be found in our [Wiki Pages](https://github.com/hydroshare/
+
+    if ($http_host != "FQDN_OR_IP") {
+
+To
+
+    if ($http_host != "FQDN_OR_IP:8080") {
+
+    Replace line 35 on the file nginx/config-files/hydroshare-local-nginx.conf.template from
+
+    rewrite ^ http://FQDN_OR_IP$request_uri permanent;
+
+To
+
+    rewrite ^ http://FQDN_OR_IP:8080$request_uri permanent;
+
+    Add new two lines into .gitignore file to make sure you will not commit your local setting to GitHub
+
+        local-dev.yml
+
+        /nginx/config-files/hydroshare-local-nginx.conf.template
+
+3. Log into Docker via application and command line.
+
+    Command line : docker login
+
+    You will be asked to enter your username and password.into 
+
+4. Launch the stack
+
+./local-dev-first-start-only.sh
+
+Following the screen instruction to continue.
+
+    Run the following command on completion to launch Hydroshare: docker-compose -f local-dev.yml up 
+
+5. Sanity Checks
+
+    Some WARNINGs are normal. 
+
+    HydroShare is available in your browser at http://localhost or http://localhost:8080  in case you are running inside a VM
+
+    The default admin page is http://localhost/admin
+
+    The default admin account is admin:default
+
+    Swagger API docs http://localhost/hsapi/
+
+6. Start & Stop & Log
+
+To start HydroShare, only need to open a windows shell, change to HydroShare code directory then run
+
+docker-compose -f local-dev.yml (up | down) [-d] [--build]
+
+Note bracketed -d for daemon is optional and you don’t paste in the brackets
+
+    Use -d option in case you want to type new command on this windows or don’t want to see real-time output log.
+
+    Use --build option in case docker keeps image in cache and does not update correctly while modifying the Dockerfile and working with PyCharm
+
+CREATE NEW ACCOUNT - This is the same as it's always been in HydroShare. Ask a teammate or hack at it. Basically open a hydroshare console window then use the UI to sign up for a new account and watch the hydroshare container console (docker logs hydroshare) for a verification link and paste that into your browser and save the new account in the UI.
+
+If your develop machine is slow, the defaultworker container may not work properly because it won’t see iRODS containers alive after timeout. Please wait seconds, open new terminal window then restart this container by the command: docker  restart  defaultworker, or just only start the whole HydroShare system by this command: ./slow-start.sh (no need to run docker-compose  -f  local-dev.yml  up) but by this way, you will not have native log window.
+
+
+
+To stop HydroShare, only need to close the running windows or open a new windows then run
+
+docker-compose -f local-dev.yml down
+
+All data is persisted for the next start.
+
+
+
+To see the logs in case you start with -d option, open a windows then run
+
+docker-compose -f local-dev.yml logs
+
+Or
+
+docker logs <container name>
+
+
+
+Warning Suppression
```

Comment:
This section about warning suppression can probably be removed as it pertained to a special circumstance for Mark and the document listed is not openly accessible.

Reply:
Removed in current commit